components/UserProvider.js

```diff
@@ -101,6 +101,10 @@ class UserProvider extends React.Component {
         this.setState({ enforceTwoFactorAuthForLoggedInUser: true });
         throw new Error(error.message);
       }
+      if (error.message.includes('Cannot use this token')) {
```

Comment:
What's that? I could not find any match.

Reply:
It matches an error message thrown by the API if you use the token on the wrong route: https://github.com/opencollective/opencollective-api/blob/d19f38226d79e6dd8f2daa65b94e33f12363f4f9/server/middleware/authentication.js#L102